test/technical.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-import Alpha from '../';
 import delay from 'delay';
-const alpha = Alpha();
+import Alpha from '../';
+const alpha = Alpha({ key: process.env.API_KEY });
```

Comment:
Lets keep the old env name

```suggestion
const alpha = Alpha({ key: process.env.AV_KEY });
```